dotnet-authserver/src/TeacherIdentity.AuthServer/Journeys/CreateUserHelper.cs

```diff
@@ -87,11 +89,6 @@ public async Task<User> CreateUserWithTrnLookup(AuthenticationState authenticati
             TrnLookupStatus = authenticationState.TrnLookupStatus
         };
 
-        if (populatePreferredName)
-        {
-            user.PreferredName = authenticationState.GetPreferredName();
```

Comment:
This did not fall back to OfficialName and has been fixed in this PR. May require running some SQL to fix any new registrations that would have used the OfficialName default